### ISSUE TITLE
Some floodlights don't turn on after restarting world

### ISSUE THREAD

**Issue by rahangar** [BluSunrize/ImmersiveEngineering#769]
I have several flood lights on my farms, each being powered by its own windmill.  If I quit the game and restart the world, some of the flood lights don't come back on.  I don't see a pattern so far, so no correlations to report as of now.  This is a bit of a pain because each time I restart my world, I have to go to each floodlight and click with the hammer, and that turns them on.

**Comment by theunkn0wn1** [comment]
By any chance are the chunks the floodlights force loaded?

**Comment by rahangar** [comment]
Yes, I use a chunk loader, and these chunks are well within the boundaries of the loader.

**Comment by theunkn0wn1** [comment]
By any chance it's ChickenChunks? See #589

**Comment by rahangar** [comment]
Yes, thanks for the pointer.  I will switch to a different chunk loader.

**Comment by codewarrior0** [comment]
I have a similar problem with floodlights. I am not using chunk loaders. A dozen floodlights around my base are powered by a full capacitor bank backed by several generators. When loading the world, some of the floodlights seem to be powered off.

I spliced a breaker switch into the single cable that connects all of my floodlights. In the "off" position, all of my floodlights turn off as expected. After switching the breaker to "on", some but not all of the floodlights come to life, and some of those that do will immediately turn off seconds later.

**Comment by cobra** [comment]
I noticed something like this as well, but could not yet find a way to trigger this behaviour reliably. *something* is definitely broken, there have been enough reports about this now.

**Comment by BluSunrize** [comment]
Please keep in mind that floodlights only work when it's dark. they won't place lightsources during the day.

**Comment by codewarrior0** [comment]
Huh, that's weird. I went over to one of the "dead" floodlights, paused the game, and then opened the world in MCEdit's read-only mode. MCEdit showed me that the "dead" light was in fact powered and flooding the area with FakeLight blocks and the lighting of the surrounding area was correct for an active floodlight. In game, the area is still dark.

**Comment by codewarrior0** [comment]
> Please keep in mind that floodlights only work when it's dark. they won't place lightsources during the day.

These are indoor floodlights.

**Comment by theunkn0wn1** [comment]
I also observed this behavior when testing for the original issue (the one
induced by ChickenChunks) but not reliably. I also observed that the only
way to get the light to emit it's fake light when affected is to hit it
with the hammer or placing a block next to it. Perhaps the client is not
receiving a block update from the server?
On Dec 13, 2015 2:31 AM, "cobra" <notifications@github.com> wrote:
>
> I noticed something like this as well, but could not yet find a way to
trigger this behaviour reliably. something is definitely broken, there have
been enough reports about this now.
